src/main/java/io/unlogged/AgentCommandExecutorImpl.java

```diff
@@ -1103,6 +1111,19 @@ private Object createParameterUsingMocking(String methodParameter, Class<?> para
         return parameterObject.getMockedFieldInstance();
     }
 
+	private boolean checkForSubClass(Class<?> fieldType) {
```

Comment:
rename the method to checkCanClassBeExtended

Reply:
yeah, changed.